target/linux/x86/base-files/lib/preinit/01_sysinfo

```diff
@@ -21,6 +21,10 @@ do_sysinfo_x86() {
 		"Supermicro:Super Server")
 			continue
 			;;
+		"PC Engines:APU")
```

Comment:
This should be move up so it's properly sorted.

Reply:
done